hyperware-ai/hyperdrive
# Drag and drop apps, settings buttons

## Problem

- [x] need a home button in system settings
- [x] widget settings modal needs a button guiding users to system settings as well
- [x] favorited apps need to be drag-and-droppable

## Testing

- Open settings app by going to widgets settings modal and clicking the new 'Looking for system settings?' button
- from the settings app, click the home button in the upper right to go back home
- on the homepage, favorite 3 apps
- drag and drop them and confirm the resulting order is what you intended
- refresh the page and confirm the order of favorited apps persists

## Notes 

- app order doesn't persist between boots as i believe that i heard we don't want the homepage app to keep any data other than in-memory
